File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/editor/EnemySpawnPointLockAcceptMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.editor;

import de.hsrm.mi.swtpro03.server.game.dtos.EnemySpawnPointDTO;
import de.hsrm.mi.swtpro03.server.game.dtos.ItemDTO;

public class EnemySpawnPointLockAcceptMessage {

	public static final String TYPE = "EnemySpawnPointLockAcceptMessage";

	private EnemySpawnPointDTO enemySpawnDTO;
	
	public EnemySpawnPointLockAcceptMessage() {
		
	}

	public EnemySpawnPointLockAcceptMessage(EnemySpawnPointDTO enemySpawnDTO) {
		this.enemySpawnDTO = enemySpawnDTO;	
	}

	public EnemySpawnPointDTO getEnemySpawnDTO() {
		return enemySpawnDTO;
	}

	public void setEnemySpawnDTO(EnemySpawnPointDTO enemySpawnDTO) {
		this.enemySpawnDTO = enemySpawnDTO;
	}

	
}

File: client/Client/Network/Messages/Editor/EnemySpawnPointLockAcceptMessage.cs
using LoadRunnerClient.DTOs;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LoadRunnerClient.Network.Messages {
	class EnemySpawnPointLockAcceptMessage {

		public const string TYPE = "EnemySpawnPointLockAcceptMessage";

		public EnemySpawnPointLockAcceptMessage() { }

		public EnemySpawnPointLockAcceptMessage(EnemySpawnPointDTO enemySpawnDTO) {
			this.enemySpawnDTO = enemySpawnDTO;
		}

		private EnemySpawnPointDTO _enemySpawnDTO;
		public EnemySpawnPointDTO enemySpawnDTO {
			get => _enemySpawnDTO;
			set => _enemySpawnDTO = value;
		}

	}
}
